docs/user_guides/fs/feature_view/helper-columns.md

```diff
@@ -11,17 +11,17 @@ Hopsworks Feature Store provides a functionality to define two types of helper c
 
 ## Inference Helper columns
 `inference_helper_columns` are a list of feature names that are not used for training the model itself but are used for extra information during online or batch inference. 
-For example computing [on-demand feature](../../../concepts/fs/feature_group/on_demand_feature.md) like distance between previous and current place of transaction `loc_delta_t_minus_1` in credit card fraud detection system.
-Feature `loc_delta_t_minus_1` will be computed using previous transaction coordinates `longitude` and `latitude` that needs to fetched from the feature store and compared to the new transaction coordinates that arrives at inference application. 
-In this use case `longitude` and `latitude` are `inference_helper_columns`. They are not used for training but are necessary for computing [on-demand feature](../../../concepts/fs/feature_group/on_demand_feature.md) `loc_delta_t_minus_1`.
+For example computing [on-demand feature](../../../concepts/fs/feature_group/on_demand_feature.md) like days left until credit card is valid at the time of transaction `days_until_valid` in credit card fraud detection system.
+Feature `days_until_valid` will be computed using credit cart valid through date `cc_valid_through` that needs to be fetched from the feature store and compared to the new transaction date that arrives at 
```

Comment:
Change `days_until_valid` to `days_valid` everywhere.

The feature `days_valid` will be computed using the credit cart expiry date that needs to be fetched from the feature store and compared to the transaction date that the transaction is performed on ( `days_valid` = `expiry_date` - `current_date`).

Reply:
`cc_valid_through`-> `expiry_date` everywhere